inputs/demand/industry/industry_useful_demand_for_chemical_other_electricity_efficiency.ad

```diff
@@ -1,10 +1,10 @@
 - query = UPDATE(V(industry_useful_demand_for_chemical_other_electricity), preset_demand, NEG(USER_INPUT()))
 - priority = 0
-- max_value = 10.0
-- min_value = -10.0
+- max_value = 100.0
+- min_value = 0.0
 - start_value = 0.0
-- step_value = 0.1
+- step_value = 1
```

Comment:
For consistency, I would denote these inputs with 1 decimal: `- step_value = 1.0`